contracts/feature-tests/use-module/tests/gov_module_whitebox_test.rs

```diff
@@ -0,0 +1,603 @@
+        |sc| {
+            sc.init_governance_module(
+                managed_token_id!(GOV_TOKEN_ID),
+                managed_biguint!(QUORUM),
+                managed_biguint!(MIN_BALANCE_PROPOSAL),
+                VOTING_DELAY_BLOCKS,
+                VOTING_PERIOD_BLOCKS,
+                LOCKING_PERIOD_BLOCKS,
+            );
+        },
+    );
+
+    world.set_state_step(SetStateStep::new().block_nonce(10));
+
+    world
+}
+
+pub fn propose(
+    world: &mut ScenarioWorld,
+    proposer: &Address,
+    gov_token_amount: u64,
+    dest_address: &Address,
+    endpoint_name: &[u8],
+    args: Vec<Vec<u8>>,
+) -> usize {
+    let use_module_whitebox =
+        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
+
+    let mut proposal_id = 0;
+
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new().from(proposer).esdt_transfer(
+            GOV_TOKEN_ID,
+            0,
+            &rust_biguint!(gov_token_amount),
+        ),
+        |sc| {
+            let mut args_managed = ManagedVec::new();
+            for arg in args {
+                args_managed.push(managed_buffer!(&arg));
+            }
+
+            let mut actions = MultiValueEncoded::new();
+            actions.push(
+                (
+                    GAS_LIMIT,
+                    managed_address!(dest_address),
+                    managed_buffer!(endpoint_name),
+                    args_managed,
+                )
+                    .into(),
+            );
+
+            proposal_id = sc.propose(managed_buffer!(b"change quorum"), actions);
+        },
+    );
+
+    proposal_id
+}
+
+#[test]
+fn test_init() {
+    setup();
+}
+
+#[test]
+fn test_change_gov_config() {
+    let mut world = setup();
+    let use_module_whitebox =
+        WhiteboxContract::new(USE_MODULE_ADDRESS_EXPR, use_module::contract_obj);
+
+    let mut current_block_nonce = 10;
+
+    let proposal_id = propose(
+        &mut world,
+        &address_expr_to_address(FIRST_USER_ADDRESS_EXPR),
+        500,
+        &address_expr_to_address(USE_MODULE_ADDRESS_EXPR),
+        b"changeQuorum",
+        vec![1_000u64.to_be_bytes().to_vec()],
+    );
+
+    assert_eq!(proposal_id, 1);
+
+    // vote too early
+    world.whitebox_call_check(
+        &use_module_whitebox,
+        ScCallStep::new()
+            .from(SECOND_USER_ADDRESS_EXPR)
+            .esdt_transfer(GOV_TOKEN_ID, 0, rust_biguint!(999))
+            .no_expect(),
+        |sc| {
+            sc.vote(proposal_id, VoteType::UpVote);
+        },
+        |r| {
+            r.assert_user_error("Proposal is not active");
+        },
+    );
+
+    current_block_nonce += VOTING_DELAY_BLOCKS;
+    world.set_state_step(SetStateStep::new().block_nonce(current_block_nonce));
+
+    world.whitebox_call(
+        &use_module_whitebox,
+        ScCallStep::new()
+            .from(SECOND_USER_ADDRESS_EXPR)
+            .esdt_transfer(GOV_TOKEN_ID, 0, rust_biguint!(999)),
```

Comment:
I think it should work without the rust_biguint! we should have direct conversion from small int